clients/ts/FunctionalTests/package.json

```diff
@@ -11,19 +11,20 @@
   },
   "devDependencies": {
     "@types/debug": "0.0.30",
-    "@types/jasmine": "^2.8.6",
-    "@types/karma": "^1.7.3",
-    "@types/node": "^9.4.6",
+    "@types/jasmine": "^2.8.8",
+    "@types/karma": "^1.7.5",
+    "@types/node": "^9.6.28",
     "debug": "^3.1.0",
-    "es6-promise": "^4.2.2",
-    "jasmine": "^3.1.0",
-    "jasmine-core": "^3.1.0",
-    "karma": "^2.0.4",
+    "es6-promise": "^4.2.4",
+    "jasmine": "^3.2.0",
+    "jasmine-core": "^3.2.1",
+    "karma": "^3.0.0",
```

Comment:
@Eilon Note: Updated dependency. Still used at test time only. I'll capture this in my work on automating registrations.